projects/igniteui-angular/src/lib/core/styles/themes/schemas/light/_action-strip.scss

```diff
@@ -0,0 +1,57 @@
+@import '../shape/action-strip';
+////
+/// @group schemas
+/// @access private
+/// @author <a href="https://github.com/desig9stein" target="_blank">Marin Popov</a>
+////
+
+/// Generates a light action-strip schema.
+/// @type {Map}
+/// @prop {Color} $color [currentColor] - The color used for the action strip component text and icons.
+/// @prop {Map} $background [igx-color: ('grays', 100] - The color used for the action strip component content background.
+/// @prop {Map} $delete-action [igx-color: ('error')] - The color used for the delete icon in action strip component.
+/// @prop {Number} border-radius [1] - The border radius fraction, between 0-1 to be used for action strip actions container.
+
```

Comment:
Check if the produced documentation includes this item. Not sure sassdoc plays nicely with empty lines.